terraform/ecs/ecs.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_ecs_cluster" "ecscluster" {
+  name = "${var.projectname}-cluster"
+}
+
+resource "aws_launch_configuration" "grp3-lc" {
```

Comment:
no need to have identification on the resource name `grp3` just call it `ecs-lc` or something like that